cmd/storj-sim/network.go

```diff
@@ -222,14 +225,47 @@ func newNetwork(flags *Flags) (*Processes, error) {
 	versioncontrol.ExecBefore["run"] = func(process *Process) error {
 		return readConfigString(&versioncontrol.Address, versioncontrol.Directory, "address")
 	}
-
 	// gateway must wait for the versioncontrol to start up
 
 	// Create satellites
 	if flags.SatelliteCount > maxInstanceCount {
 		return nil, fmt.Errorf("exceeded the max instance count of %d with Satellite count of %d", maxInstanceCount, flags.SatelliteCount)
 	}
 
+	// set up redis servers
+	var redisServers []*Process
+	var redisPort string
```

Comment:
Hmm, why is this here? There are multiple redis server and each of them has a separate port.

Reply:
oh for the actual port value like 10014

Reply:
Yes, but there will be multiple of them. You can rely on `port(satellitePeer, redisTCP)` for the time being everywhere. Or `redisServers[i].Address`

Reply:
hmm I'm not sure I follow, the way I have it right now each satellite will use a separate redis port numbered 100X4, which was generated by the `port` method. I keep track of them using `ports[]`

Reply:
`port` is idempotent and so is `redisServers[i].Address` that way you don't need to have a separate `[]string` to track them.

Reply:
oh I see, the github history is misleading, I thought you were implying all satellites could use the same redis server/port.

Reply:
I still haven't had my morning coffee to properly explain things :D

Reply:
:D no worries at all, it's past midnight here so I'm misreading things lol